Android/app/build.gradle

```diff
@@ -36,9 +36,23 @@ android {
     }
 
     buildTypes {
+        debug{
+            def credsFile = rootProject.file("secure\\debug_creds.properties")
```

Comment:
Does this backslash work on non-Windows systems?

Reply:
> Does this backslash work on non-Windows systems?

It works on Mac, I can't vouch for anything else 😅

Reply:
If it works on Macs, that's good enough for me.